.github/workflows/run-tests.yml

```diff
@@ -1,6 +1,11 @@
 name: run-tests
 
-on: [push, pull_request]
+on:
+    push:
+        branches:
+            - main
```

Comment:
Should still be changed to master instead of main

Reply:
haha oops, fixed!